public/app/dashboard/profileController.js

```diff
@@ -125,6 +125,22 @@
     };
 
     /**
+     * Toggle activity breakdown selection (contributions or impacts).
+     *
+     * @param  {String}         filter          Which portion of the breakdown to display
+     * @return {void}
+     */
+    var toggleBreakdown = $scope.toggleBreakdown = function(filter) {
+      $scope.breakdown.selected = filter;
+
+      // Track toggle.
+      analyticsService.track('Toggle breakdown filter', {
```

Comment:
@johncrossman For event names, I'm going off the specs in https://docs.google.com/document/d/1WqwvMP2HHvhkKAQsDsoxv4trfLR_OSXu7h90fVHI01Q - which are, yes, a little vague in their descriptions and could probably use a round of tightening and specificity once we're verified the mechanics are working.